packages/app-store/sylapsvideo/lib/VideoApiAdapter.ts

```diff
@@ -21,7 +21,7 @@ const SylapsApiAdapter = (): VideoApiAdapter => {
     deleteMeeting: async (uid: string): Promise<void> => {
       Promise.resolve();
     },
-    updateMeeting: (bookingRef: PartialReference, event: CalendarEvent): Promise<VideoCallData> => {
+    updateMeeting: (bookingRef: PartialReference): Promise<VideoCallData> => {
```

Comment:
@hariombalhara The warning you fixed i think was fixed by 046d5e952339f37c6193154c741da7ea66a41cf0
But if you dont like unsued i understant.

Reply:
We may want to alto remove CalendarEvent import i guess.